internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -617,9 +622,20 @@ func convertTopic(t manifest.Topic, accountID, region, app, env, svc string) (*t
 		Name:           t.Name,
 		AllowedWorkers: t.AllowedWorkers,
 		AccountID:      accountID,
+		Partition:      partition,
 		Region:         region,
 		App:            app,
 		Env:            env,
 		Svc:            svc,
 	}, nil
 }
+
+// getSNSTopicPartition returns the partition of the SNS topic, up to but not including the actual topic name,
+// for the current service
+func getSNSTopicPartition(region string) (string, error) {
```

Comment:
I don't think this necessarily needs its own function; can we either remove this, or rename it to 
```go
// getPartition returns the partition associated with the given region
getPartition(region string) (string, error) {
``` 
and also use it in `internal/pkg/deploy/cloudformation/stack/app.go` when we get the partition there?